query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -136,7 +136,9 @@ pub fn row_value_to_prisma_value(
             ParameterizedValue::Integer(i) => {
                 PrismaValue::Float(Decimal::from_f64(i as f64).expect("f64 was not a Decimal."))
             }
-            ParameterizedValue::Text(s) => PrismaValue::Float(s.parse().unwrap()),
+            ParameterizedValue::Text(_) | ParameterizedValue::Bytes(_) => {
+                PrismaValue::Float(p_value.as_str().unwrap().parse().unwrap())
```

Comment:
A necessary "Yo dawg! I heard you like unwrapping" meme...

Reply:
Oh, missed that one - we can probably do better error handling there.